### PR TITLE
Add Stop method to VideoPlayer

### PR DESCRIPTION
**Pull request by kuan9611**
Fix issue #1345 
Add a Stop method to VideoPlayer that stops it from playing and rewinds the video view back to the beginning. VideoPlayer version number is bumped.
Also simplify the logic in FullScreenVideoUtil to target Eclair or higher versions of Android.

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

**Comment by AppInventorWorkerBee** [comment]
All tests passed. +1 Verified.

File: appinventor/components/src/com/google/appinventor/components/runtime/VideoPlayer.java [hunk 1]
```diff
@@ -283,6 +283,16 @@ public void Pause() {
     }
   }
 
+  @SimpleFunction(
+      description = "Resets to start of video and pauses it if video was playing.")
+  public void Stop() {
+    Log.i("VideoPlayer", "Calling Stop");
+    // Call start() to ensure frame shown is updated
```
[the remark this answers is not in the document]
Comment (kuan9611) [reply appinventor/components/src/com/google/appinventor/components/runtime/VideoPlayer.java]: Comment updated. This behavior is also noted in the description of SeekTo.

**Comment by AppInventorWorkerBee** [comment]
All tests passed. +1 Verified.

**Comment by AppInventorWorkerBee** [comment]
All tests passed. +1 Verified.

**Pull request merged**